locale/en/docs/guides/backpressuring-in-streams.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Backpressuring in Streams
+layout: docs.hbs
+---
+
+# Backpressuring in Streams
+ 
+The purpose of this guide will describe what backpressure is and the 
+problems in Streams that it solves. The first part of this guide will cover 
+these questions. The second part of the guide will introduce suggested best 
+practices to ensure your application's code is safe and optimized.
+
+We assume a little familiarity with the general definition of 
+[`backpressure`][], [`Buffer`][] and [`EventEmitters`][] in Node.js, as well as 
+some experience with [`Streams`][]. If you haven't read through those docs, 
+it's not a bad idea to take a look at the API documentation first, as it will 
+help expand your understanding while reading this guide and for following along 
+with examples.
+
+## The Problem with Streams
```

Comment:
Syntactically, I am still a bit confused. Wikipedia describes backpressure in IT as

> the build-up of data behind an I/O switch if the buffers are full and incapable of receiving any more data; the transmitting device halts the sending of data packets until the buffers have been emptied and are once more capable of storing information. 

In this instance .. is backpressure the problem or the I/O switch that streams implements?